lxd/device/nic_bridged.go

```diff
@@ -564,12 +571,29 @@ func (d *nicBridged) setFilters() (err error) {
 		}
 	}
 
-	// Retrieve existing IPs, or allocate new ones if needed.
-	IPv4, IPv6, err := d.allocateFilterIPs()
-	if err != nil {
+	// Check if the parent is managed and load config. If parent is unmanaged continue anyway.
+	var IPv4, IPv6 net.IP
+	_, netInfo, err := d.state.Cluster.NetworkGet(d.config["parent"])
+	if err != nil && err != db.ErrNoSuchObject {
 		return err
 	}
 
+	// If parent bridge is unmanaged check that IP filtering isn't enabled.
+	if err == db.ErrNoSuchObject || netInfo == nil {
+		if shared.IsTrue(d.config["security.ipv4_filtering"]) || shared.IsTrue(d.config["security.ipv6_filtering"]) {
+			return fmt.Errorf("IP filtering requires using a managed parent bridge")
```

Comment:
It does? I thought we had made it possible to use both of those so long as you set a static ipv4.address/ipv6.address.

I'm reasonably sure that we actually rely on this working for Travis-CI.

Reply:
@stgraber the fix we added for Travis was a managed bridge without DHCP.